.github/workflows/pr-advisor.yml

```
@@ -219,3 +219,245 @@ jobs:
+          echo "--- changed files ---"
+          cat changed.txt
+
+      # ADVISORY ON PURPOSE, AND ONLY FOR NOW. A brand-new guard that hard-fails
+      # every PR on day one is how guards get switched off permanently, so the
+      # first landing cannot block. The verdict is posted either way; what
+      # `continue-on-error` buys is that a bug in ME never stops the owner.
+      - name: Which lane is this PR in
+        id: classify
+        if: steps.rules.outputs.ready == 'true'
+        continue-on-error: true
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2046:warning:6:44: Quote this to prevent word splitting

(shellcheck)